src/pal/tools/gen-buildsys-gcc.sh

```diff
@@ -37,21 +37,30 @@ elif command -v "${gcc_prefix}gcc$2$3" > /dev/null
 elif command -v "${gcc_prefix}gcc-$2$3" > /dev/null
     then
         desired_gcc_version="-$2$3"
-elif command -v ${gcc_prefix}gcc > /dev/null
+elif command -v "${gcc_prefix}gcc" > /dev/null
     then
         desired_gcc_version=
 else
-    echo "Unable to find $gcc_prefixgcc Compiler"
+    echo "Unable to find ${gcc_prefix}gcc Compiler"
     exit 1
 fi
 
-if [ -z "$CC" ]; then
-  export CC="$(command -v ${gcc_prefix}gcc$desired_gcc_version)"
+if [ -z "$CLR_CC" ]; then
+    CC="$(command -v "${gcc_prefix}gcc$desired_gcc_version")"
+    gcc_link="$(command -v link)"
+else
+    CC="$CLR_CC"
+    gcc_link="$CC"
```

Comment:
Use $CLR_LINK here to be consistent. I can match CLR_LINK to CLR_CC in my environment variables. Somebody might need the cross compiled link library some day.

Reply:
That's better. I have aligned gcc_link with gcc_ar and rest of the utilities.

```sh
CLR_LINK=/path/to/custom/linker ./build.sh x64 checked gcc   # shall use custom linker
./build.sh x64 checked gcc # shall use default linker
```